cli/README.md

```diff
@@ -157,7 +163,7 @@ OPTIONS:
     -r, --remappings <remappings>...                         the remappings
         --remappings-env <remappings-env>                     [env: DAPP_REMAPPINGS=]
         --root <root>
-            the project's root path, default being the current directory [default: std::env::current_dir().unwrap()]
+            the project's root path, default being the current directory
```

Comment:
```suggestion
            the project's root path, default being the current working directory
```

Reply:
Wouldn't this be the project directory rather than `$pwd`?

Reply:
Nope, you were right, this is how it works now.